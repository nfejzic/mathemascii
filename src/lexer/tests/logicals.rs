use super::Snapshot;

super::test_snap!(and_or_not, "andornot and or not");
super::test_snap!(implications, "implies <=> => iff");
super::test_snap!(quantors, "AA EE forallexists AAEE");
super::test_snap!(other, "_|_ bottop TT |-- |==");
