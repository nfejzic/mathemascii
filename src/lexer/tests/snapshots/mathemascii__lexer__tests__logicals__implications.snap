---
source: src/lexer/tests/logicals.rs
expression: "Snapshot((input, tokens))"
---
implies <=> => iff

implies
^^^^^^^ -> Logical(Implies) at: 0 -> 7

        <=>
        ^^^ -> Logical(IfAndOnlyIf) at: 8 -> 11

            =>
            ^^ -> Logical(Implies) at: 12 -> 14

               iff
               ^^^ -> Logical(IfAndOnlyIf) at: 15 -> 18


